### DISCUSSION TITLE
segmentation fault

### DISCUSSION THREAD

**Discussion by maximatt** [openai/whisper#2114] [General]
Hi. I'm trying to make a test with whisper, but I receive a segmentation fault in all cases that I tried:

This is most basic test that I made:

```
$ pyenv virtualenv 3.10.6 test
$ pip install openai-whisper
$ whisper test.wav --language Spanish
```
test wav file is created as  `ffmpeg -i 01-video_01.mp4 -ar 16000 -ac 1 test.wav`

Any idea?
Thanks in advance!

**Comment by glangford** [comment]
Have you tried
`whisper 01-video_01.mp4 --language es`

**Reply by maximatt** [reply]
yes., I tried it... but with same results (even if I execute `whisper test.wav`, `whisper test.wav --language es`, etc)

But I have things to try.. 1) recheck the prerequisites (example: install rust), 2) try with another test file, 3) etc
I guess that is something in my environment... since that feels weird that a package with such bug has been published.

Thanks a lot for your time (I will keep you updated)
cheers!

**Comment by maximatt** [comment]
Hi to everyone

Well.. was my fault. After installed rust [1], whisper works like a charm 

Thanks a lot!!!

[1]: `$ curl https://sh.rustup.rs -sSf | sh -s -- -y`
